### ISSUE TITLE
Add Conda Feedstock

### ISSUE THREAD

**Issue by marcelteraflow** [ianhi/jupyterlab-vimrc#26]
For those who use the conda build system, is there a **conda-forge** feedstock?

**Comment by ianhi** [comment]
Apparently I started the process back in march and it never got through: https://github.com/conda-forge/staged-recipes/pull/18223

I think i'll reopen and ping. Thanks!

**Comment by ianhi** [comment]
Ah it looks as though the build failed and I didn't know why. If it fails again I'm now even further removed from understanding building pyhton x typescript packages so here's hoping it's not too tricky to solve

**Comment by marcelteraflow** [comment]
https://github.com/conda-forge/staged-recipes/pull/21385

**Comment by marcelteraflow** [comment]
Please see my attempt. I'm willing to amend my PR to make you maintainer.

**Comment by marcelteraflow** [comment]
This package is amazing and I'm tired of having it be the only package under "install with pip" section in my `environment.yml`

**Comment by marcelteraflow** [comment]
Perhaps a good idea would be to add a `meta.yml` to the repo so it stays up to date? I'm not sure whether my pull request will stay up to date with the pypi version.

**Comment by ianhi** [comment]
thanks for taking the initiative! the build still fails unfortunately, likely because it's been a long time since I updated the build scripts for this package, and they are significantly out of date with the modern jupyterlab-extension build system. It should be updated to follow the current state of https://github.com/jupyterlab/extension-cookiecutter-ts

**Comment by ianhi** [comment]
> Please see my attempt. I'm willing to amend my PR to make you maintainer.

yes please

> Perhaps a good idea would be to add a meta.yml to the repo so it stays up to date? I'm not sure whether my pull request will stay up to date with the pypi version.

Maybe I'm misunderstanding your concern, but once the PR is accepted it will be moved to a diffrent conda forge repo and a bot will autodetect pushes to pypi and update conda-forge.

**Comment by ianhi** [comment]
re the build system, I have experiments to run today, but towards the end of the week iwll likely have some time to look into it. Of course if you feel particularly inspired I'd also be happy to accept PRs that successfully update to the new system.

**Comment by marcelteraflow** [comment]
> yes please

https://github.com/conda-forge/staged-recipes/pull/21385/commits/9bc1a06dd4cbd7250a8d96544ac37edf13bc5e14

**Comment by marcelteraflow** [comment]
Yeah I'm also still getting build errors. As you can conclude I'm a heavy user of the conda infrastructure but haven't maintained packages myself.

I'll leave my pull request for now and let you focus on yours.

I checked the [build logs for linux](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=620757&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd)

Don't know if you have tried to comb through the linux64 **raw logs**?
